includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -0,0 +1,277 @@
+							'key' => 'global-navigation-brandlink-vertical-games'
+						],
+						'href' => $this->getHref( 'games' ),
+					],
+					[
+						'type' => 'link-branded',
+						'brand' => 'movies',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-navigation-brandlink-vertical-movies'
+						],
+						'href' => $this->getHref( 'movies' ),
+					]
+				]
+			],
+			'wikis' => [
+				'header' => [
+					'type' => 'line-text',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'global-navigation-wikis',
+					],
+				],
+				'links' => [
+					[
+						'type' => 'link-text',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-navigation-brandlink-wikis-explore'
+						],
+						'href' => $this->getHref( 'explore-wikis' ),
+					],
+					[
+						'type' => 'link-text',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-navigation-wikis-communitycentral'
+						],
+						'href' => $this->getHref( 'community-central' ),
+					],
+					[
+						'type' => 'link-text',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-navigation-wikis-fandomuniversity'
+						],
+						'href' => $this->getHref( 'fandom-university' ),
+					]
+				]
+			],
+			'search' => [
+				'module' => [
+					'type' => 'search',
+					'results' => [
+						'url' => 'http://starwars.wikia.com/wiki/Special:Search?fulltext=Search',
+						'param' => 'query'
+					],
+					'suggestions' => [
+						'url' => 'http://starwars.wikia.com/index.php?action=ajax&rs=getLinkSuggest&format=json',
+						'param' => 'query'
+					],
+					'placeholder-inactive' => [
+						'type' => 'translatable-text',
+						'key' => 'global-navigation-search-placeholder-inactive'
+					],
+					'placeholder-active' => [
+						'type' => 'translatable-text',
+						'key' => 'global-navigation-search-placeholder-active'
+					]
+				]
+			],
+			'create_wiki' => [
+				'header' => [
+					'type' => 'link-text',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'wikia-create-wiki-link-start-wikia'
+					],
+					'href' => $this->getPageUrl( 'CreateNewWiki', NS_SPECIAL ),
+				]
+			]
+		];
+
+		if ($wgUser->isLoggedIn()) {
+			$data['user'] = $this->getLoggedInUserData( $wgUser->getId() );
+			$data['notifications'] = $this->getNotifications( $wgUser->getId() );
+		} else {
+			$data['anon'] = $this->getAnonUserData();
+		}
+
+		return $data;
+	}
+
+	private function getHref( $hrefKey ) {
+		return $this->hrefs[$this->lang][$hrefKey] ?? $this->hrefs['default'][$hrefKey];
+	}
+
+	private function getAnonUserData() {
+		return [
+			'header' => [
+				'type' => 'link-image',
+				'href' => '#',
+				'image' => 'wds-icons-user',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-navigation-userinfo-anon-avatar-title',
+				],
+			],
+			'links' => [
+				[
+					'type' => 'authentication-link',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "authentication-link" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)